src/Exception/InvalidArgumentException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mdtt\Exception;
+
+class InvalidArgumentException extends \Exception
+{
+
+}
```

Comment:
@sadeesh-sdet, PHP already has this: https://www.php.net/manual/en/class.invalidargumentexception.php. What is the benefit of us creating our own? Do you want to add additional features to this exception in some way?
- If yes, then I suggest renaming the class to match your intention. If someone sees this in code, it will lead to confusion. Also, extend `\InvalidArgumentException` rather than `\Exception`. This is to make this catchable in the same hierarchy.
- If no, but you still want to use our own exception, then extend `\InvalidArgumentException` at least as mentioned above.
- If no, and there's no reason to use our own class, then delete this file and use `\InvalidArgumentException` in the other files.

Reply:
Thanks @hussainweb for the detailed clarification. Addressed in this recent [commit](https://github.com/sadeesh-sdet/mdtt/commit/5a5b5adf3f3a732497d07ac232d10ec76a754d87).